redfish-core/lib/license_service.hpp

```diff
@@ -0,0 +1,693 @@
+#pragma once
+
+#include "redfish_util.hpp"
+#include "registries.hpp"
+
+#include <app.hpp>
+#include <error_messages.hpp>
+#include <registries/privilege_registry.hpp>
+#include <utils/json_utils.hpp>
+
+#include <filesystem>
+#include <string_view>
+#include <variant>
+
+namespace redfish
+{
+namespace messages
+{
+static void addMessageToErrorJson(nlohmann::json& target,
```

Comment:
Why is this in this file?

Reply:
this is to make future rebase easier. We have less files to worry about in future. May be we have to make it a public  API in upstream first. But never know when that will get accepted. Till then lets just duplicate it to lessen the rebase effort.

Reply:
I don't see how this saving us much effort but 🤷 I guess I am fine with this

Reply:
Shouldn't this be done in `error_messages.cpp`?

Reply:
yes but that makes it one more files to consider for the future rebase. There was a proposal to put everything in single file so that the  future rebase will be just a single file copy.